packages/core-plugin-api/src/plugin/types.ts

```diff
@@ -88,6 +98,8 @@ export type PluginConfig<
   routes?: Routes;
   externalRoutes?: ExternalRoutes;
   featureFlags?: PluginFeatureFlagConfig[];
+  options?: PluginOptions;
```

Comment:
No need for this on this type right?

Reply:
It is. I'm using it everything in the codebase as the place to keep the options.
![Screenshot 2022-07-06 at 11 13 44](https://user-images.githubusercontent.com/2265094/177515486-7892c4be-cd66-47bf-9e3e-c87ed5ee1c1c.png)

Reply:
Yep we should make those options private to the plugin instance instead. The config does not need them

Reply:
Made it private